.circleci/config.yml

```diff
@@ -0,0 +1,57 @@
+version: 2.1
+
+executors:
+  python38:
+    docker:
+      - image: circleci/python:3.8
+
+commands:
+  setup_tox:
+    description: "Install tox"
+    steps:
+      - run: pip install -U tox virtualenv tox-factor
```

Comment:
It [seems like](https://github.com/tox-dev/tox/blob/master/setup.cfg#L45) `virtualenv` is already a `tox` dependency.

Reply:
will fix in #828

Reply:
Fixed, turns out tox is also a dependency of tox-factor